packages/components/src/List/ListComposition/SortBy/SortBy.component.js

```diff
@@ -34,45 +68,61 @@ function SortBy(props) {
 	};
 
 	// Sort field
-	const onSortByChange = (val, event) => performChange(event, { ...currentValue, sortBy: val });
+	const onSortByChange = (val, event) => {
+		performChange(event, { ...currentValue, sortBy: val });
+	};
 
 	// Sort order
 	const onOrderChange = event =>
-		performChange(event, { ...currentValue, isDescending: !currentValue.isDescending });
+		performChange(event, { ...currentValue, isDescending: !isDescending });
 
 	return (
-		<React.Fragment>
+		<Nav className={theme('tc-sort-by')}>
 			<Navbar.Text>
-				<label htmlFor={id}>{t('LIST_TOOLBAR_SORT_BY', { defaultValue: 'Sort by:' })}</label>
+				<label className={theme('tc-sort-by-label')} htmlFor={id}>
+					{t('LIST_TOOLBAR_SORT_BY', { defaultValue: 'Sort by:' })}
+				</label>
 			</Navbar.Text>
 			<Nav>
-				<NavDropdown
-					id={id}
-					title={selectedLabel}
-					onSelect={onSortByChange}
-					aria-label={t('LIST_CHANGE_DISPLAY_MODE', {
-						defaultValue: 'Change sorting option. Current sorting: {{sortBy}}.',
-						sortBy: selectedLabel,
-					})}
-				>
-					{options.map(({ key, label }) => (
-						<MenuItem id={`${id}-${key}`} key={key} eventKey={key} aria-label={label}>
-							{label}
-						</MenuItem>
-					))}
-				</NavDropdown>
-				<NavItem
-					id={`${id}-order`}
-					onClick={onOrderChange}
-					aria-label={t('LIST_CHANGE_SORT_BY_ORDER', {
-						defaultValue: 'Change sort order. Current order: {{sortOrder}}.',
-						sortOrder: orderLabel,
-					})}
-				>
-					{orderLabel}
-				</NavItem>
+				{options.length === 1 ? (
+					<li className="navbar-text">{options[0].name}</li>
```

Comment:
Is this acceptable to have an `<li />` tag being a child of a `<Nav />` component ? Even though `<Nav />` might wrap or not its children with `<ul/>`

Reply:
To be honest I copy paste the sort by code from legacy list

Reply:
If I run an audit on the page with lighhouse I don't have any warning about this.

Reply:
I made some tweak, w3c validator wasn't happy with a part.